ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/Wei.java

```diff
@@ -23,6 +23,7 @@
 public final class Wei extends BaseUInt256Value<Wei> {
 
   public static final Wei ZERO = of(0);
+  public static final Wei NO_REWARD = of(UInt256.MAX_VALUE);
```

Comment:
This feels like the wrong place for this constant - maybe `BlockProcessor` instead?

Reply:
I can move it there.  It also needs to be visible to AbstractBlockCreator and MainnetBlockProcessor, but it looks like the needed dependencies are in place.